lib/interactor.ex

```diff
@@ -28,40 +117,12 @@ defmodule Interactor do
     defp alias_multi, do: nil
   end
 
-  defp define_call do
+  defp define_callback_defaults do
     quote do
-      def call(input) do
-        Interactor.call(__MODULE__, input, @_repo)
-      end
-
       def before_call(c), do: c
       def after_call(r), do: r
```

Comment:
I hadn't considered it until now... might be something interesting to rubber duck